components/camel-aws/camel-aws2-kinesis/src/main/java/org/apache/camel/component/aws2/kinesis/Kinesis2Configuration.java

```diff
@@ -47,6 +51,9 @@ public class Kinesis2Configuration implements Cloneable {
     @UriParam(description = "Amazon Kinesis client to use for all requests for this endpoint")
     @Metadata(label = "advanced", autowired = true)
     private KinesisClient amazonKinesisClient;
+    @UriParam(description = "Amazon Kinesis async client to use for all requests for this endpoint that require an async client")
```

Comment:
If it's specific to KCL is better to add the info to the description

Reply:
No worries, I'll add a better description for this param.

Reply:
Thanks. I'm also curious about your use case, are you using this in production? What version? Did you see space for more improvements?

Reply:
I'm currently evaluating this for a use case in production. It has been working pretty well so far, but I am still yet to do some serious load testing, which will probably happen after New Years. One possible improvement I thought of was around potentially injecting a custom AWS credential provider implementation and make use of the library client instantiation. It was a more involved patch, so for now I went with the current approach which also works.